compiler/src/Reporting/Error/Syntax.hs

```diff
@@ -70,6 +70,7 @@ data Error
   | NoPortModulesInPackage A.Region
   | NoEffectsOutsideKernel A.Region
   | ParseError Module
+  deriving (Show)
```

Comment:
Show instances in this file aren't actually needed for anything yet, but they were helpful when debugging.

Reply:
Yeah. My bar for adding `Show` is pretty low. I spend enough time debugging that it feels I'm often adding these all the time.

Leave it in 👍